impeller/entity/contents/filters/gaussian_blur_filter_contents.h

```diff
@@ -117,7 +116,7 @@ class GaussianBlurFilterContents final : public FilterContents {
   const Vector2 sigma_ = Vector2(0.0, 0.0);
   const Entity::TileMode tile_mode_;
   const BlurStyle mask_blur_style_;
-  std::shared_ptr<Geometry> mask_geometry_;
```

Comment:
This is eschewed  by the C++ style guide: https://google.github.io/styleguide/cppguide.html#Ownership_and_Smart_Pointers

Who is the owner of this Geometry?